CMakeLists.txt

```diff
@@ -110,6 +110,16 @@ add_subdirectory(deps)
 # Add the C++ sources subdirectory
 add_subdirectory(gympp)
 
+find_package(Eigen3)
+find_package(iDynTree)
+
+# Add the controllers
+if(${Eigen3_FOUND} AND ${iDynTree_FOUND})
```

Comment:
Not strictly a problem, but auto-detection of the dependency to use is not really packaging-friendly (what is compiled by the project depends on the enviroment :/), and quite error prone, i.e. if fore some reason or bug `find_package(iDynTree)` fails, as a user I prefer a clear configuration error at CMake time, not that I am able to compile and install fine `gym-ignition`, and only when I try to use it (perhaps after a while) I discover the controller part is not installed. What sometimes we do in other projects is to use auto-detection to provide the default values of the `<package>_USES_<deps>` options, but observe the options if the users set it, and print a clear error if it is not possible to observe them.

Reply:
I agree, thanks for the feedback. Addressed in 0124287a73990951ad46b0859f645f24f7389542.